### ISSUE TITLE
System RAM not enough, what can I do to decrease memory cost ?

### ISSUE THREAD

**Issue by YYZZAA** [guanyingc/PS-FCN#5]
Hello, thanks a lot for your code!
When I am testing your pre-trained model on dataset DiLiGenT, about 14G RAM is taken up, which makes my machine crash down . I guess it is because that too many pictures are loaded at the same time. So I'm here to ask how can I modify something in the code such as batch size to decrease the number of pictures loaded at the same time to lower the RAM occupation and is it possible to lower the memory cost to about 10G while testing your pre-trained model? 
Looking forward to your reply

**Comment by guanyingc** [comment]
Hi, sorry for the late reply. Are you using CPU for testing? In fact, at test time, we only use a single batch. However, there are 96 images in each patch. To save memory, you can crop the object of interest in the image using object mask first. Please refer to https://github.com/guanyingc/SDPS-Net/blob/master/scripts/cropDiLiGenTData.py for reference.

**Comment by YYZZAA** [comment]
Ok, I'll try it out, thanks a lot! 
P.S. In fact, I'm using GPU on google colab for testing and the RAM showed here got filled up and then the program crashed.
![image](https://user-images.githubusercontent.com/53884269/79177886-5eadbe80-7e36-11ea-9e07-523aef349ddc.png)

**Comment by guanyingc** [comment]
I remember PS-FCN used less than 8 GB memory using GPU. Not sure what is the problem. But you can try cropping first.
